recipes/at-spi2-atk/all/conanfile.py

```diff
@@ -0,0 +1,74 @@
+from conans import ConanFile, Meson, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+import shutil
+
+
+class AtSPI2AtkConan(ConanFile):
+    name = "at-spi2-atk"
+    description = "library that bridges ATK to At-Spi2 D-Bus service."
+    topics = ("conan", "atk", "accessibility")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://gitlab.gnome.org/GNOME/at-spi2-atk"
+    license = "LGPL-2.1-or-later"
+    generators = "pkg_config"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        }
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
```

Comment:
```suggestion

    def config_options(self):
        if self.settings.os == "Windows":
            del self.options.fPIC

```

Reply:
this package is linux-only, because it depends on at-si2-core, which is linux only

Reply:
Therefore, wouldn't it be better to raise also in this recipe if not linux, instead of relying on its dependency to do so?